### PR TITLE
Markets list mobile cleanup

### PR DESCRIPTION
**Pull request by andrewdamelio**
TODO
- few mobile fixes

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
This pull request has been automatically deployed to [FeaturePeek](https://featurepeek.com). 🌈

 This link will always point to this PR's latest changes: https://peek.run/4kghjn4

File: packages/augur-ui/src/modules/app/components/app.styles.less [hunk 1]
```diff
@@ -8,6 +8,8 @@ html,
 
 body {
   background: @color-primary-background;
+  top: 0px !important;
```
Comment (bthaile) [inline packages/augur-ui/src/modules/app/components/app.styles.less:12]: can we get away from `!important`

File: packages/augur-ui/src/modules/common/constants.ts [hunk 2]
```diff
@@ -232,6 +232,19 @@ export enum MARKET_SORT_PARAMS {
...
 }
 
+export const SORT_OPTIONS = [
+  { value: MARKET_SORT_PARAMS.LIQUIDITY, header: 'Highest liquidity' },
+  { value: MARKET_SORT_PARAMS.OPEN_INTEREST, header: 'Highest open interest' },
+  { value: MARKET_SORT_PARAMS.VOLUME, header: 'Highest volume' },
+  { value: MARKET_SORT_PARAMS.CREATION_TIME, header: 'Recently created' },
+  { value: MARKET_SORT_PARAMS.END_DATE, header: 'Ending soon ' },
+  { value: MARKET_SORT_PARAMS.RECENTLY_TRADED, header: 'Recently Traded' },
+  {
```
Comment (bthaile) [inline packages/augur-ui/src/modules/common/constants.ts]: this one can be removed. There's a clean up ticket to do so. but might as well remove here.

**Review by bthaile — CHANGES_REQUESTED** [review]
remove`!important` could cause issues down the road

**Review by bthaile — APPROVED** [review]
lgtm

**Pull request merged**